app/services/work_packages/update_service.rb

```diff
@@ -108,6 +109,12 @@ def move_time_entries(work_packages, project_id)
       .update_all(project_id:)
   end
 
+  def move_work_package_memberships(work_packages, project_id)
+    Member
+      .where(entity: work_packages)
+      .update_all(project_id:)
+  end
+
```

Comment:
:+1:

The contracts for members should also check that if there is an entity having a project_id, then that value is the value for the member's project_id as well. Can also be part of a subsequent PR, though.